lib/learn_to_rank/ranker_status.rb

```diff
@@ -0,0 +1,103 @@
+require "aws-sdk-sagemaker"
+require "httparty"
+
+module LearnToRank
+  class RankerStatus
+    attr_reader :errors
+
+    def initialize(timeout: 0.1)
+      @errors = [] # Strings
+      @timeout = timeout
+      @healthy = check_health
+    end
+
+    def healthy?
+      @healthy && errors.none?
+    end
+
+  private
+
+    attr_reader :timeout
+
+    GOOD_STATES = %w(AVAILABLE InService Updating Creating SystemUpdating RollingBack).freeze
```

Comment:
We might want to think more about the sagemaker states - eg, `RollingBack` could be cause for a warning, as it means something's gone wrong with the current model.

Reply:
OK. I've removed those AWS states for now, I'll re-add some later when I check Sagemaker health.